src/lib/transition_frontier/transition_frontier.ml

```diff
@@ -63,36 +69,38 @@ struct
     let empty = Ledger_builder.Aux.empty ~parallelism_log_2:4
 
     module Diff = struct
-      type nonrec t = Ledger_builder_diff.t
+      type nonrec t = ledger_diff
 
       let of_ledger_builder_diff = Fn.id
 
       let to_ledger_builder_diff = Fn.id
     end
   end
 
+  type staged_ledger = Ledger_builder.t
+
   (* Right now, Staged_ledger is a thin wrapper over Ledger_builder *)
   module Staged_ledger : sig
-    type t
+    type t = staged_ledger
 
     val create :
          transaction_snark_scan_state:Transaction_snark_scan_state.t
       -> ledger:Ledger.Mask.Attached.t
       -> t Or_error.t
 
-    val transaction_snark_scan_state : t -> Transaction_snark_scan_state.t
+    val _transaction_snark_scan_state : t -> Transaction_snark_scan_state.t
 
     val ledger : t -> Ledger.Mask.Attached.t
 
     val apply :
          t
-      -> Transaction_snark_scan_state.Diff.t
       -> logger:Logger.t
+      -> Transaction_snark_scan_state.Diff.t
       -> t Or_error.t
   end = struct
     (* TODO *)
```

Comment:
This TODO is kinda annoying. It does not specify what I need TODO. Can you remove it?